Recommend using machine users for automated tasks (#1157)
platformsh/platformsh-docs: src/configuration/routes/https.md

            -----END CERTIFICATE-----
```

### Automated SSL certificate renewal using Cron

> If the Let's Encrypt certificate is due to expire in less than one month it will automatically renew at that time.  That makes it feasible to set up auto-renewal of the Let's Encrypt certificate.  The caveat is that, like any deploy, there is a very brief downtime (a few seconds, usually) so it's best to do during off-hours.

Reviewer: ```suggestion
If the Let's Encrypt certificate is due to expire in less than one month then it will be renewed automatically during a deployment.  That makes it feasible to set up regular auto-renewal of the Let's Encrypt certificate.  The caveat is that, like any deploy, there is a very brief downtime (a few seconds, usually) so it's best to do during off-hours.
```

> **note**
>

> > Automated SSL certificate renewal using cron requires to [get an API token and install the CLI in your application container](/gettingstarted/cli/api-tokens.md).

Reviewer: "requires to get" doesn't work, it needs to be "requires you to get" or "requires getting [and installing]"
